README.md

```diff
@@ -1 +1,122 @@
+
+- [📖 About the Project](#about-project)
+- [🛠 Built With](#built-with)
+- [Tech Stack](#tech-stack)
+- [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [👥 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderbord_Project] <a name="about-project"></a>
+
+**[Leaderbord_Project]** The "leaderboard" website displays scores submitted by different players. It also allows you to submit your score. All data is preserved
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- An HTML File
+- A CSS File
+- A Javascript file
+- ES6
+- A modules file
+- An md file
+- Webpack
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming soon!!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+A Browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+`https://github.com/Ouail-01/Leaderboard_Project.git`
+
+### Install
+
+Install this project with:
+A commandline interface e.g Gitbash
+
+### Usage
+
+To run the project, execute the following command:
+
+To run the project, execute the following command:
+Click on the live server button on your IDE
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Ouail Laarabi**
+
+- GitHub: [@Ouail-01](https://github.com/Ouail-01?tab=overview&from=2023-01-01&to=2023-01-02)
+- Twitter: [@Ouail_laarabi](https://twitter.com/Ouail_Laarabi)
+- LinkedIn: [Ouail Laarabi](https://www.linkedin.com/in/ouail-laarabi-53203b250/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Future Features <a name="future-features"></a>
+
+**In future features, I intend to include more pages such as:**
+
+- Create a collection that keeps the score.
+- Add a functionality that allows users the score at leaderboard
+- Display all saved score in the collection on the page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+<p align="right"><p align="right">(<a href="#readme-top"><a href="#readme-top">back to top</a></a>)</p></p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed.
```

Comment:
- [ ] @Ouail-01 broken links hurt the user's experience and make the developer looks unprofessional. Kindly fix this link and make sure it's working.